ethportal-peertest/src/jsonrpc.rs

```diff
@@ -0,0 +1,131 @@
+use std::io::prelude::*;
+use std::os::unix::net::UnixStream;
+use std::slice::Iter;
+
+use hyper::{self, Body, Client, Method, Request};
+use log::info;
+use serde_json::{self, Value};
+
+use trin_core::portalnet::U256;
+
+#[derive(Copy, Clone)]
+pub struct JsonRpcEndpoint {
+    pub method: &'static str,
+    pub id: &'static u8,
```

Comment:
Is the `id` just a convenience field so that we do not need to generate a unique `id` for each JSON-RPC request that we test?

Reply:
Yup, exactly. Not strictly necessary to do here, but since `params` will also soon be added to the struct, seemed harmless to stick these in there as well